### ISSUE TITLE
Change Vertical Scrollbar Width

### ISSUE THREAD

**Issue by sankayop** [VSCodium/vscodium#247]
**Describe the bug**
To be able to modify the vertical scrollbar width, I could use https://marketplace.visualstudio.com/items?itemName=be5invis.vscode-custom-css#review-details
but then I would need to modify files (thus the rights) inside /usr/lib/codium.
Unfortunately, codium is providing binaries only.
Question: How to modify the css layout of codium without having access to the lib?

**Please confirm that this problem is VSCodium-specific**
- [v] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.
I installed Microsoft's Visual Studio Code and after modifying the rights to /usr/lib/code, I could modify the CSS succesfully.

**Please confirm that the issue/resolution isn't already documented**
- [v] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
0. Install following extension: ext install be5invis.vscode-custom-css
1. Create and go to 'custom.css', put following code inside:
"
/*tab group scrollbar;*/
.monaco-workbench>.part.editor>.content .editor-group-container>.title.tabs>.tabs-and-actions-container>.monaco-scrollable-element .scrollbar,
.monaco-workbench>.part.editor>.content .editor-group-container>.title.tabs>.tabs-and-actions-container>.monaco-scrollable-element .scrollbar .slider {
    height: 10px !important;
}
"
2. Add "    
"vscode_custom_css.imports": ["file:///pathToYourLocation/.vscode/custom.css"],
    "vscode_custom_css.policy": true
" to Settings.json
3. Press F1 and "Reload Custom CSS and JS"
4. Restart

**Expected behavior**
Check the tab scrollbar height. Should be greater.

**Screenshots**
![image](https://user-images.githubusercontent.com/23059127/63591333-be591000-c5ae-11e9-868d-17165426614e.png)

**Desktop (please complete the following information):**
 - OS: Manjaro
 - Architecture x64
 - Version 5.1.21-1
Linux 5.1.21-1-MANJARO #1 SMP PREEMPT Sun Jul 28 09:33:43 UTC 2019 x86_64 GNU/Linux

**Additional context**
- I do appreciate the work being done on vs-codium and wish it would not bother me that much but I do work with a lot of tabs and scrolling between them is fastidious because of the really thin scrollbar... Thanks for all your effort anyway and sorry for the annoying request...
- I forgot to add the request that was done for Visual Studio Code: 
https://stackoverflow.com/questions/53943278/visual-studio-code-expand-the-horizontal-bar-for-scrolling-tabs
https://github.com/Microsoft/vscode/issues/32192#issuecomment-450222349

**Comment by stripedpajamas** [comment]
Hi @sankayop 👋

Trying to understand what the difference is between Code and Codium here. They are both providing binaries, right? Can you explain more about what the experience is with Code vs what it is with Codium?

**Comment by sankayop** [comment]
Hi Stripedpajamas,
for codium, it is a bin version but for code not:
"""
community/code 1.37.1-2 (8.1 MiB 63.6 MiB) 
    The Open Source build of Visual Studio Code (vscode) editor
"""
I don't know what to say else except that for Code, because it's not only a bin, i do have the requested folders.
Hope it helps.

**Comment by stripedpajamas** [comment]
Can you share how you installed Visual Studio Code vs how you installed VSCodium?

**Comment by sankayop** [comment]
Hi Peter,
I'm sorry for the late reply.
I use manjaro (archlinux distro) together with yay.

Therefore, I installed code-1.37.1-2 by just typing "yay -S code" and he
does the make install stuff.
Below the link to the package:
$ yay -Si code
Repository      : community
Name            : code
Version         : 1.37.1-2
Description     : The Open Source build of Visual Studio Code (vscode)
editor
Architecture    : x86_64
URL             : https://github.com/microsoft/vscode
Licenses        : MIT
Groups          : None
Provides        : None
Depends On      : electron4  libsecret  libx11  libxkbfile  ripgrep
Optional Deps   : bash-completion: Bash completions
                  zsh-completions: ZSH completitons
Conflicts With  : None
Replaces        : None
Download Size   : 8.09 MiB
Installed Size  : 63.62 MiB
Packager        : Filipe Laíns <lains@archlinux.org>
Build Date      : Mon 19 Aug 2019 09:32:54 CEST
Validated By    : MD5 Sum  SHA-256 Sum  Signature

for vscodium, I typed "yay -S vscodium-bin"
Below the link to the package:
$yay -Si vscodium-bin
Repository      : aur
Name            : vscodium-bin
Keywords        : None
Version         : 1.38.1-1
Description     : Binary releases of VS Code without MS
branding/telemetry/licensing.
URL             : https://github.com/VSCodium/vscodium
AUR URL         : https://aur.archlinux.org/packages/vscodium-bin
Groups          : None
Licenses        : MIT
Provides        : code
Depends On      : fontconfig  libxtst  gtk3  python  cairo  alsa-lib  nss
 gcc-libs  libnotify  libxss  gconf  glibc>=2.28-4
Make Deps       : None
Check Deps      : None
Optional Deps   : gvfs  libdbusmenu-glib
Conflicts With  : None
Maintainer      : plague-doctor
Votes           : 46
Popularity      : 8.530370
First Submitted : Fri 19 Apr 2019 01:11:21 AM CEST
Last Modified   : Fri 13 Sep 2019 12:19:07 AM CEST
Out-of-date     : No

So in other words, for vscode, because i compile it from source code, I do
have all the required folder to do the custom modifications where it is not
the case for vscodium since I only have the binary version.

Hope it helps,
Yoshiyuki

**Comment by stripedpajamas** [comment]
Now I see. @CRKatri can you chime in here? I'm not familiar with what the substantive differences are between Microsoft's `code` package and our `vscodium-bin` package.

**Comment by CRKatri** [comment]
So taking a quick look at the PKGBUILDS, `code` is the Arch Linux Maintainers compiling vscode from source, although based on this quick assessment it still contains telemetry(?) but also has https://open-vsx.org, whereas vscodium doesn't have the telemetry. I'm not one hundred percent sure that's accurate but it should be close enough. [Here is the PKGBUILD and patches that Arch uses.](https://github.com/archlinux/svntogit-community/tree/packages/code/trunk)

**Comment by CRKatri** [comment]
Also, Arch/Manjaro are binary based distributions so unless he is compiling it himself using `makepkg` he is using binaries, just changing who's binaries.
